README.md

```diff
@@ -103,6 +104,30 @@ $ fdiff [OPTIONS] [PRE-FONT FILE URL] [POST-FONT FILE FILE PATH]
 
 ⭐ **Tip**: Remote git repository hosting services (like Github) support access to files on different git branches by URL.  Use these repository branch URL to compare fonts across git branches in your repository.
 
+#### As Git's diff driver for fonts
+
+Git can be configured to automatically use a specific tool to diff specific file types. These are the steps to use `fdiff` as the default diff output for font files.
+
+1. Tell Git how to run the tool. As with most git configuration options this may be set for a repository (`--local`), for your user (`--global`), or system (`--system`). We recommend setting this at the user level. Assuming `fdiff` is available in your path, this setting should do the trick:
+
+        git config --global diff.fdiff.command 'fdiff -c --git'
+
+    This will write it to your `$HOME/.gitconfig` file looking like this:
+
+    ```gitconfig
+    [diff "fdiff"]
+        command = fdiff -c --git
```

Comment:
I suggest that we add instructions for a local configuration file too.  I needed to install on the path `.git/config` from the root of the repository to make this work

Reply:
Done, I amended the commit that added this with some more versatile instructions.